### ISSUE TITLE
[Bug] [Bug] [ ERROR ] Exception occurred during running replacer "REPLACEMENT_ID" (<class 'extensions.middle.PartialInfer.PartialInfer'>): Stopped shape/value propagation at "StatefulPartitionedCall/model_1/stem_conv/Conv2D" node.

### ISSUE THREAD

**Issue by MiraGaikwad** [openvinotoolkit/openvino#12884]
System information (version)
OpenVINO => 2021.4.2
Operating System / Platform => Windows 64 Bit
Compiler => Visual Studio 2017
Problem classification: Model Conversion
Framework: TensorFlow 2.8.0
Model name: EfficientNetV2S
Detailed description
I have trained the model in google Colab and saved file in h5 format.
I then frozen that model.
Now trying to convert that mode into IR format
but I am facing below error -
[ ERROR ] Cannot infer shapes or values for node "StatefulPartitionedCall/model_1/stem_conv/Conv2D".
[ ERROR ] index 3 is out of bounds for axis 0 with size 3
[ ERROR ]
[ ERROR ] It can happen due to bug in custom shape infer function <function Convolution.infer at 0x000001E97F619D08>.
[ ERROR ] Or because the node inputs have incorrect values/shapes.
[ ERROR ] Or because input shapes are incorrect (embedded to the model or passed via --input_shape).
[ ERROR ] Run Model Optimizer with --log_level=DEBUG for more information.
[ ERROR ] Exception occurred during running replacer "REPLACEMENT_ID" (<class 'extensions.middle.PartialInfer.PartialInfer'>): Stopped shape/value propagation at "StatefulPartitionedCall/model_1/stem_conv/Conv2D" node.

Steps to reproduce
Please find the H5 file can be found in attachment

https://drive.google.com/file/d/15Js2rughGyaAI55SGk0sS0iLKoHotb29/view?usp=sharing

I am using below code to convert model into .pb format.
import tensorflow as tf
model = tf.keras.models.load_model('EfficientNetV2S_New_baseline.h5')
tf.saved_model.save(model,'saved_model')

Using below command to convert model into IR format where I am facing above error
mo --saved_model_dir saved_model --data_type FP16 --output_dir IR_model_FP16 --input_shape [1,224,224,3]

Could you please help over this ?

**Comment by andrei-kochin** [comment]
Hello @MiraGaikwad,

Thank you for reaching OpenVINO!
While we are looking into this could you please let me know is it possible for you to try latest OpenVINO 2022.1 and try running ModelOptimizer without specifying input_shape option?

Thank you!

**Comment by rkazants** [comment]
@MiraGaikwad, I've requested the access to the model. Please provide it.

Thanks,
Roman

**Comment by MiraGaikwad** [comment]
@rkazants - I have now given the access. 

@andrei-kochin - Yeah, I already tried giving input_shape option but it did not work.

**Comment by MiraGaikwad** [comment]
@rkazants - I am able to convert that using OPENVINO 2022.1 version but as I want to also use it on Raspberry Pi, it requires OPENVINO 2021 version where it is giving error while conversion.

**Comment by andrei-kochin** [comment]
@MiraGaikwad  @rkazants  I also able to get it converted successfully:

`> mo --saved_model_dir C:\Users\akochin\Downloads\saved_model\saved_model\ --silent
[ SUCCESS ] Generated IR version 11 model.
[ SUCCESS ] XML file: C:\Users\akochin\saved_model.xml
[ SUCCESS ] BIN file: C:\Users\akochin\saved_model.bin
[ SUCCESS ] Total execution time: 172.01 seconds.

`
@MiraGaikwad What's wrong with 2022.1 for Raspberry Pi?

**Comment by MiraGaikwad** [comment]
@andrei-kochin - I am able to convert in 2022.1 but not with 2021 version.

I am using 2021 on Raspberry Pi 3 because I only found 2021 distribution for Raspberry Pi 3 on this site - https://download.01.org/opencv/2021/openvinotoolkit/2021.2/

**Comment by andrei-kochin** [comment]
Hello @MiraGaikwad,

We still provide support for building from source [Link](https://github.com/openvinotoolkit/openvino_contrib/wiki/How-to-build-ARM-CPU-plugin)

Could you please try with the provided instruction to get 2022.1 or latest master?

**Comment by andrei-kochin** [comment]
Hello @MiraGaikwad,

do you have any great news for us? Were your build attempt successfull?

**Comment by MiraGaikwad** [comment]
@andrei-kochin  - Unfortunately no the reason being that model I was at-least able to convert using OPENVINO 2022.1 version. There is new model I have trained which I am not able to convert using even 2022 version.

Please find the model link below - 
https://colab.research.google.com/drive/17fCQz7YZtSnuK8NcK5SDiNJ7g_HIqBR4?authuser=1

It is giving error as - 

[ FRAMEWORK ERROR ]  Cannot load input model: OOM when allocating tensor with shape[3,3,256,256] and type float on /job:localhost/replica:0/task:0/device:CPU:0 by allocator cpu
         [[{{node RestoreV2}}]]

**Comment by andrei-kochin** [comment]
Hello @MiraGaikwad,

Can we help you somehow else in scope of this issue?
If not please consider closing this issue.

Thank you!

**Comment by avitial** [comment]
@MiraGaikwad you can find the v2022.2 pre-built Debian package available for download in [Releases/2022.2.0](https://github.com/openvinotoolkit/openvino/releases/tag/2022.2.0) or [storage.openvinotoolkit.org](https://storage.openvinotoolkit.org/repositories/openvino/packages/2022.2/linux/). 

For 2022.1 version you might need to build from source if such version is needed, as the pre-built Debian package isn't available for download for such release. Find build instructions for Myriad plugin [here](https://github.com/openvinotoolkit/openvino/wiki/BuildingForRaspbianStretchOS) or for ARM CPU plugin [here](https://github.com/openvinotoolkit/openvino_contrib/wiki/How-to-build-ARM-CPU-plugin).

Closing this. Feel free to reopen and provide additional information or ask any questions related to this topic.
